docs/maatregelen/4-owk-06-optimaliseer-AI-training.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Optimaliseer AI-trainingsprocessen voor energie-efficiëntie
```

Comment:
algoritme trainingsprocessen? Volgens mij in het bijzonder relevant bij AI maar ook nuttig voor niet AI-toepassingen?

Reply:
Valt te betwisten denk ik: zijn er algoritmen die niet-AI zijn maar wel een trainingsproces hebben? Alle algoritmes die je echt traint, stellen dus gewichten/parameters af en leren patronen aan, en zou je dus AI kunnen noemen.

Of loop ik dan niet goed gelijk met de definitie van AI zoals de AI-verordening stelt, @BartdeVisser?

Reply:
Dit is uiteraard hét lastige punt. Ons uitgangspunt is dat al deze gevallen onder AI-systeem vallen:
- supervised learning
- unsupervised learning
- reinforcement learning
- deep learning

Zijn er denkbare gevallen bij supervised learning waarin we denken dat het geen AI is? Voldoende mate van autonomie? Als het antwoord hierop nee is dan staat het denk ik goed.

Reply:
Ja ik zou van mening zijn dat algoritmes die niet onder die categorieën vallen, en dus geen AI zijn, ook geen training behoeven, of in elk geval geen heftige training waar je energie bij kan besparen. Maar vind het ook lastig ja! Na nog even rondzoeken of je ook trainingsprocessen kan hebben als het een simpel algoritme is, denk ik dat we het gewoon zo moeten laten staan.